chain-signatures/contract-eth/scripts/responder.js

```diff
@@ -0,0 +1,53 @@
+const hre = require("hardhat");
+
+async function main() {
```

Comment:
Do you want to keep this after you add a real indexer to the main system?

Reply:
No, it is reference client before integration with node is done